config/config.yaml

```diff
@@ -13,6 +13,8 @@ internalApi:
     enabled: true
 
 adapters:
+  redis:
+    poolSize: 500
```

Comment:
What makes me think about this is a VERY SPECIFIC way to say the max number of schedulers is 500 (and non-straightforward). Is there a way we could improve what this config var says in practical terms?

Reply:
Not sure if I agree with you, technically this is not the max number of schedulers, you could have 500 schedulers with a lower number of connections pool, since theoretically, a scheduler will always be taking and releasing a connection to perform its actions. Besides that, this configuration is EXACTLY the `redis client connections pool size` and I think we should be clear about what it is to prevent misconfiguration, not sure how we could be more precise with the naming here, do you have any suggestions?